Aztec/Classes/NSAttributedString/Conversions/AttributedStringSerializer.swift

```diff
@@ -105,8 +105,9 @@ class AttributedStringSerializer {
         let childAttributes = self.attributes(for: element, inheriting: attributes)
         let content = NSMutableAttributedString()
 
-        if let representation = implicitRepresentation(for: element, inheriting: childAttributes) {
-            content.append(representation)
+        if let inserter = inserter(for: element) {
+            let insertedString = inserter.insert(for: element, inheritedAttributes: childAttributes)
```

Comment:
This is a great line to try and bring up something that feels off to me.

If you read the line as if it was english text, you'll notice that the "inserter" name is not very appropriate for the new entity we're creating.  Mostly because no insertion takes place at all here, but rather a "conversion" or "construction" of a string, based on an element.

I think we should consider other naming options that better represent what's happening here.

I know it's mostly on my for pushing the concept of a new `Inserter` entity, so please bear with me @jklausa.

My best suggestion right now would be something like `Converter`... which would make this code become:

```swift
if let converter = converter(for: element) {
    let string = converter.convert(element, inheritingAttributes: chldAttributes)
} else {
   // etc
}
```